maestro/src/agents/agent.py

```diff
@@ -17,22 +18,36 @@ def __init__(self, agent:dict) -> None:
             agent_name (str): The name of the agent.
         """
         # TODO: Review which attributes belong in base class vs subclasses
-        self.agent_name = agent["metadata"]["name"]
-        self.agent_framework = agent["spec"]["framework"]
-        self.agent_model = agent["spec"]["model"]
-
-        self.agent_desc = agent["spec"]["description"]
-        self.agent_instr = agent["spec"]["instructions"]
-        self.agent_input = agent["spec"].get("input")
-        self.agent_output = agent["spec"].get("output")
-        self.instructions = f"{self.agent_instr} Input is expected in format: {self.agent_input}" if self.agent_input else self.agent_instr
-        self.instructions = f"{self.instructions} Output must be in format: {self.agent_output}" if self.agent_output else self.instructions
-
+        self.agent_name = agent['metadata']['name']
```

Comment:
Let's open an new issue that make consistent these to the schema.

Reply:
This was @maximilien original commit 5240be732e19892940fecfa4ab475e3d974ab078 - are you ok to open an issue for the followon activity?

Reply:
I believe only @akihikokuroda and I have merge privileges... as others make more contributions we will revisit and grant. Thank you.